transport/nio/pom.xml

```diff
@@ -123,6 +133,34 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-checkstyle-plugin</artifactId>
             </plugin>
+<!-- Enable to build benchmarks.jar after there is new version of plugin
```

Comment:
@jfallows will create a PR against maven plugins. Until then, just run in the IDE.